src/engine/node/node.render.ts

```diff
@@ -143,6 +169,10 @@ export function updateLocalNodeResources(
         node.mesh = undefined;
       }
 
+      // TODO: skinned mesh disposal
```

Comment:
Hmm, yeah we need to look into what needs to be disposed here. If you don't do it in this PR make sure to create an issue so that we look into it soon.